styleguide/source/_patterns/03-organisms/current-issue-teaser.twig

```diff
@@ -2,7 +2,7 @@
   <div class="joe__issue-teaser--left">
```

Comment:
Is the month / current issue text supposed to be on the same line as the issue title? 
![image](https://user-images.githubusercontent.com/394763/35952934-fe594afc-0c47-11e8-8f3d-d6a88ec5ce3e.png)